packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -113,6 +113,22 @@ public function get_declarations() {
 		return $this->declarations;
 	}
 
+	/**
+	 * Filters a CSS property + value pair.
+	 *
+	 * @param string $property The CSS property.
+	 * @param string $value    The value to be filtered.
+	 * @param string $spacer   The spacer between the colon and the value. Defaults to an empty string.
+	 *
+	 * @return string The filtered declaration as a single string.
+	 */
+	protected static function filter_declaration( $property, $value, $spacer = '' ) {
+		if ( isset( $property ) && isset( $value ) && '' === trim( $spacer ) ) {
```

Comment:
Why do we need the `'' === trim( $spacer )` check here? 🤔

Reply:
I think I was overthinking this one, it was to ensure that the spacer is just whitespace and not arbitrary characters, but since we're not passing user entered values to `$spacer` I think we can ditch the check. I'll update.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/43004/commits/2260debf9016217ad20af9d77e813211186fa90f